metrics/ci_integrations/test/common/helper/file_helper_test.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:ci_integration/common/helper/file_helper.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("FileHelper", () {
+    const nonExistingFilePath = 'test/common/resources/notafile.txt';
+    const existingFilePath = 'test/common/resources/file.txt';
+    final fileHelper = FileHelper();
```

Comment:
Let's try to avoid accessing File System in Unit tests as file system is usually slow and source of additional issues.